OHCRN/platform
# 👽 💄  #742 1/2: change register page GET /invites request to client-side

## Description of Changes

This PR updates the Register page to make client-side requests to the consent API proxy to request an invite if an `inviteId` was provided in the URL.

### Consent UI
- Change `RegisterFormWrapper` to be a client component & add API fetching
- Move translations to nearest server component
- Add `params` & `disabled` to `consentApiProxyFetch` for GET invites request
- Remove unused files for server-side requests

## Testing
- Go to `/en/invite` & make sure the browser console is open. 
- Create an invite.
- Get `inviteId` from console.
- Go to `/en/register?inviteId=MY_INVITE_ID`

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [ ] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [ ] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files
